stories/va-text-input.stories.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { getWebComponentDocs, StoryDocs, propStructure } from './wc-helpers';
+
+const textInputDocs = getWebComponentDocs('va-text-input');
+
+export default {
+  title: 'Components/va-text-input',
+  parameters: {
+    docs: {
+      /* eslint-disable-next-line react/display-name */
+      page: () => <StoryDocs docs={textInputDocs.docs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  name: 'my-input',
+  label: 'My input',
+  autocomplete: false,
+  'disable-analytics': true,
+  // Fun fact: This should really be false, but that passes required="false" to
+  // the web component, which correctly interprets this as a truthy value.
+  // See also:
+  // https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute
+  required: null,
```

Comment:
I think we can make some changes so that `required=false` will behave as the value would suggest. Is this specific to the CSS?

Reply:
Well, the web component will see `this.required` as true, so it'll add the `span.required`. Not just CSS, no.

Reply:
The trouble here is that Storybook adds the attribute to the `<va-text-input>` in the DOM when it shouldn't, according to that WHATWG spec.

Reply:
Oh, is this just a storybook problem? In other JSX would we be able to have `required={false}` and have it be ok?

Reply:
Yeah, exactly. `required={false}` is okay, but `required="false"` is wrong.